samwise.js

```diff
@@ -8,74 +8,75 @@
 
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?